ui/desktop/app/styles/app.scss

```diff
@@ -665,23 +665,29 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
 
 // Settings
 .settings-card {
+  margin-bottom: 1rem;
+
   .hds-card__container {
     padding: 1rem;
+    margin-bottom: 1rem;
   }
 
   .header-and-icon {
     display: flex;
-    align-items: center;
-    gap: 0.5rem;
+    align-items: flex-start;
+    gap: 1rem;
   }
 
   .content {
     padding-left: 2.2rem;
   }
 
   .app-information {
-    display: flex;
-    justify-content: space-between;
+    display: grid;
     margin: 1rem 2rem 1rem 0;
+    grid-template-columns: repeat(3, 1fr);
```

Comment:
I changed this back to grid because with flex, I had to set width to control the space between columns and it didn't match with other rows. seemed simpler with grid, also works when we have two or three columns.

Reply:
I think this works right now but flexbox offers you the flexibility to not _have_ to be in a multiple of the exact `fr` you defined (i.e. it takes up `1fr` or `2fr`, etc). For example, I see that we'll have a server card with a URL that could actually take up two thirds of the card most likely, we'd now need extra css to have that element span two `fr` to appear correct while flex requires no extra css for that.

Instead of width, I'd recommend using `flex-basis` so that these elements can take up more space than a column width if needed

Reply:
thanks for the flex-basis reminder, I changed it to flex